cwf/gateway/fabfile.py

```diff
@@ -465,6 +465,8 @@ def _run_integ_tests(
 
     # QOS take a while to run. Increasing the timeout to 50m
     go_test_cmd = "gotestsum --format=standard-verbose "
+    # Retry once on failure
+    go_test_cmd += "--rerun-fails=1 --packages='./...' "
```

Comment:
what is this?

Reply:
it re-runs any failed tests

Reply:
*at most once. so if it's --rerun-fails=5 it'll re-run until success at most 5 times

** it'll also only run tests that failed not the whole suite

Reply:
@uri200 ^^

Reply:
ya, not sure is that is very good... we shouldnt be failing
But i guess we dont have any other option :(

Reply:
@uri200 yeah.. not ideal
This will only help with flaky tests on CI. But if we want to make integ tests mandatory, this will help.